pages/docs/modules/exec.mdx

```diff
@@ -0,0 +1,355 @@
+[`exec.command`](#type-command).
+
+```go copy filename="Example"
+>>> exec("date")
+exec.result(pid: 13289)
+>>> exec("date").stdout
+"Tue Jan  9 09:21:04 PM CET 2024\n"
+>>> exec("date", ["-Iseconds", "--utc"]).stdout
+"2024-01-09T20:22:43+00:00\n"
+>>> exec("date", ["-Iseconds"], {"env": {"TZ": "America/New_York"}}).stdout
+"2024-01-09T15:24:57-05:00\n"
+```
+
+Available configuration options for `params`:
+
+```json
+{
+  "stdout": os.stdout, // type: io.Writer
+  "stderr": os.stderr, // type: io.Writer
+  "stdin": os.stdin, // type: io.Reader | string | byte_slice
+  "dir": "/foo/bar", // working directory
+  "env": {
+    // values must be string
+    "MY_VAR": "123",
+    "TZ": "Europe/Berlin"
+  }
+}
+```
+
+## Type command
+
+Returned from the function with the same name: [`exec.command()`](#command).
+
+### command.path
+
+```go filename="Field signature"
+command.path string
+```
+
+Path of the command to run.
+
+```go copy filename="Example"
+>>> cmd := exec.command("bash")
+>>> cmd.path
+"/usr/bin/bash"
+```
+
+### command.dir
+
+```go filename="Field signature"
+command.dir string
+```
+
+Dir specifies the working directory of the command.
+If dir is the empty string, Run runs the command in the calling process's
+current directory.
+
+```go copy filename="Example"
+>>> cmd := exec.command("bash", "-c", "pwd")
+>>> cmd.dir = "/tmp"
+>>> cmd.run()
+>>> cmd.stdout
+"/tmp\n"
+```
+
+### command.env
+
+```go filename="Field signature"
+command.env list
+```
+
+Env specifies the environment of the process.
+Each entry is of the form `"key=value"`.
+If `env` is nil, the new process uses the current process's environment.
+If `env` contains duplicate environment keys, only the last value in the list
+for each duplicate key is used.
+
+As a special case on Windows, `SYSTEMROOT` is always added if missing and not
+explicitly set to the empty string.
+
+```go copy filename="Example"
+>>> cmd := exec.command("bash", "-c", "echo $MY_VAR")
+>>> cmd.env = ["MY_VAR=1234"]
+>>> cmd.run()
+>>> cmd.stdout
+"1234\n"
+```
+
+### command.stdout
+
+```go filename="Field signature"
+command.stdout io.Writer
+```
+
+The process' standard output pipe. Defaults to a buffer.
+
+If read after process has been executed, and the value was left as default,
+then the field will return a string.
+
+```go copy filename="Example"
+>>> cmd := exec.command("bash", "-c", "echo hello stdout; echo hello stderr >&2")
+>>> cmd.run()
+>>> cmd.stdout
+"hello stdout\n"
+```
+
+### command.stderr
+
+```go filename="Field signature"
+command.stderr io.Writer
+```
+
+The process' standard error pipe. Defaults to a buffer.
+
+If read after process has been executed, and the value was left as default,
+then the field will return a string.
+
+```go copy filename="Example"
+>>> cmd := exec.command("bash", "-c", "echo hello stdout; echo hello stderr >&2")
+>>> cmd.run()
+>>> cmd.stderr
+"hello stderr\n"
+```
+
+### command.run
+
+```go filename="Method signature"
+command.run()
```

Comment:
I changed it to "Method signature" here (compared to "Function signature"). Appropriate?

Reply:
Yes